spec/support/fixtures/test/main.swift

```diff
@@ -1,3 +1,23 @@
-enum UnusedEnum {
-    case unused
+protocol RedundantProtocol {
 }
+
+class SomeClass: RedundantProtocol {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/manicmaniac/danger-periphery/blob/eebb39add10edcb8769f633ddd123892cc2e1bfa/spec/support/fixtures/test/main.swift#L4"></span>Protocol 'RedundantProtocol' conformance is redundant</td>
    </tr>
  </tbody>
</table>